contrib/win32/openssh/install-sshd.ps1

```diff
@@ -1,4 +1,6 @@
-﻿# @manojampalam - authored initial script
+﻿#requires -runasadministrator
```

Comment:
Does this work with windows powershell on windows 7?

Reply:
Tried this on windows 7. It didn't work..

![image](https://user-images.githubusercontent.com/23668037/87346642-c5f70f80-c506-11ea-9812-a0a0ba52ca52.png)


Adding the below lines works.

If (!([bool]([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole] "Administrator")))
  {Throw "You must be running as an administrator, please restart as administrator"}